airbyte-cdk/python/unit_tests/sources/file_based/scenarios/csv_scenarios.py

```diff
@@ -2305,7 +2307,6 @@
             {
                 "data": {
                     "col1": "2",
-                    "col2": None,
```

Comment:
why does the expected output change?

Reply:
All Serializers have `omit_none=True` parameter that is applied recursively. Thus, all `None` values are excluded from output.
This is expected behaviour and does not break anything in protocol.